Automattic/wp-calypso
# Added plan information steps

#### Proposed Changes

* Added new plan information steps:

	<img width="1211" alt="image" src="https://user-images.githubusercontent.com/1044309/191361304-91772fc9-4c42-4a9c-83fa-b77bbed70a05.png">


- On small screens, it should be displayed with a horizontal scroll (validated with @vinimotaa):

	<img width="482" alt="image" src="https://user-images.githubusercontent.com/1044309/191361434-849319fb-f587-4479-b1fa-da4cf0bcf02c.png">



#### Testing Instructions

- After checkout this branch, open the new plugins plan page: http://calypso.localhost:3000/plugins/plans/{site}?flags=plugins-plans-page
- Make sure the current implementation math the design and works as expected

#### Pre-merge Checklist



Related to #67977
